server/http.go

```diff
@@ -269,13 +280,23 @@ type startMeetingRequest struct {
 func (p *Plugin) postMeeting(creator *model.User, meetingID int, channelID string, topic string) (*model.Post, *model.AppError) {
 
 	meetingURL := p.getMeetingURL(meetingID)
+	if topic == "" {
+		topic = "Zoom Meeting"
+	}
+
+	slackAttachment := model.SlackAttachment{
+		Fallback: fmt.Sprintf("Video Meeting started at [%d](%s).\n\n[Join Meeting](%s)", meetingID, meetingURL, meetingURL),
+		Title:    topic,
+		Text:     fmt.Sprintf("Personal Meeting ID (PMI) : [%d](%s)\n\n[:movie_camera:  Join Meeting](%s)", meetingID, meetingURL, meetingURL),
+	}
 
 	post := &model.Post{
 		UserId:    creator.Id,
 		ChannelId: channelID,
-		Message:   fmt.Sprintf("Meeting started at %s.", meetingURL),
+		Message:   fmt.Sprintf("%s has started a meeting", creator.Username),
```

Comment:
Do you mind updating this message to be the same as the one of the custom post? (i.e. `I have started a meeting`)

Reply:
Done